stdpopsim/catalog/homo_sapiens.py

```diff
@@ -880,94 +880,96 @@ def __init__(self):
             # LBK / EEF merges into Loschbour
             msprime.MassMigration(
-                time=t_LBK_Losch, source=2, destination=4,
+                time=t_LBK_Losch, source=1, destination=3,
                 proportion=1.0),
             # MA1 merges into Loschbour
             msprime.MassMigration(
-                time=t_MA1_Losch, source=5, destination=4,
+                time=t_MA1_Losch, source=4, destination=3,
                 proportion=1.0),
             # Han merges into Loschbour
             msprime.MassMigration(
-                time=t_Han_Losch, source=6, destination=4,
+                time=t_Han_Losch, source=5, destination=3,
                 proportion=1.0),
             # Change in population size in Han-Losch ancestral pop
             msprime.PopulationParametersChange(
                 time=t_Han_Losch, initial_size=N_Han_Losch,
-                population_id=4),
+                population_id=3),
             # UstIshim merges into Loschbour
             msprime.MassMigration(
-                time=t_Ust_Losch, source=7, destination=4,
+                time=t_Ust_Losch, source=6, destination=3,
                 proportion=1.0),
             # Loschbour / Non-Africans receive admixture from Neanderthals
             msprime.MassMigration(
-                time=t_Nean_to_Eurasian, source=4, destination=8,
+                time=t_Nean_to_Eurasian, source=3, destination=7,
                 proportion=p_Nean_to_Eurasian),
             # Basal Eurasians merge into Loschbour / Non-Africans
             msprime.MassMigration(
-                time=t_Basal_Losch, source=1, destination=4,
+                time=t_Basal_Losch, source=8, destination=3,
                 proportion=1.0),
             # Mbuti merge into Loschbour / Non-Africans
             msprime.MassMigration(
-                time=t_Mbu_Losch, source=0, destination=4,
+                time=t_Mbu_Losch, source=0, destination=3,
                 proportion=1.0),
             # Change in population size in Mbuti-Losch ancestral pop
             msprime.PopulationParametersChange(
                 time=t_Mbu_Losch, initial_size=N_Mbu_Losch,
-                population_id=4),
+                population_id=3),
             # Neanderthal change population size
             msprime.PopulationParametersChange(
                 time=t_NeaPopSizeChange, initial_size=N_Nean_Losch,
-                population_id=8),
+                population_id=7),
             # Neanderthal merge into Loschbour / modern humans
             msprime.MassMigration(
-                time=t_Nean_Losch, source=8, destination=4,
+                time=t_Nean_Losch, source=7, destination=3,
                 proportion=1.0),
             # Ancestral hominin population size change
             msprime.PopulationParametersChange(
                 time=t_Nean_Losch, initial_size=N_Nean_Losch,
-                population_id=4),
+                population_id=3),
```

Comment:
hey @ndukler there are a lot of tweaks to this model in this PR. i think we should probably trigger a QC step here. Rather than have an independent implementation, maybe it would be enough to compare simply summaries of variation between the old and new versions?

Reply:
Also shows that hard-coding these population IDs is problematic. What if we have short variable names for these? That would make it easier to read and verify that we're doing the right thing.

Reply:
agreed

Reply:
The QC for this model hasn't actually been completed so they could just change the population indexes for their initial run through.